nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/LatencyTrackerComputation.java

```diff
@@ -127,6 +123,26 @@ public void processTimer(ComputationContext context, String key, long timestamp)
         }
     }
 
+    protected List<LogPartitionGroup> getLogGroup() {
+        if (logGroups.isEmpty() || refreshGroup()) {
+            logGroups.clear();
+            logNames.forEach(name -> {
+                for (String group : manager.listConsumerGroups(name)) {
+                    logGroups.add(new LogPartitionGroup(group, name, 0));
+                }
+            });
+            if (verbose) {
+                log.info("Update list of consumers: " + Arrays.toString(logGroups.toArray()));
```

Comment:
Do you really need `Arrays.toString(logGroups.toArray())`?
In which case you could use a lambda with Log4j 2.

Or just `log.info("Update list of consumers: {}", logGroups)` if it does the job.

Reply:
Arrays.toString(myList.toArray()) is a common way to get a string representation of the content of a list, and necessary when using log4j 1.

Reply:
indeed just printing logGroups should be enough, will clean that soon, thx